content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,233 @@
+============
+Product type
+============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Define *product types* in Odoo to track products in varying levels of detail.
+
+Classify products as *storable* to track stock counts, allowing users to trigger :doc:`reordering
+rules <../product_replenishment/reordering_rules>` for generating purchase orders. *Consumable*
+products are assumed to always be in stock, and *service* products are performed and served by the
+business.
+
+.. seealso::
+   `Odoo Tutorials: Product Type <https://www.youtube.com/watch?v=l6j0ZkP5mLM>`_
+
+Set product type
+================
+
+To set a product type, go to :menuselection:`Inventory app --> Products --> Product` and selecting
+the desired product.
+
+On the product form, in the :guilabel:`Product Type` field, select:
+
+- :guilabel:`Storable` for products tracked with stock counts. Only storable products can trigger
+  reordering rules for generating purchase orders
+
+.. tip::
+   Choose :guilabel:`Storable` if it is necessary to track a product's stock at various locations,
+   inventory valuation, or lots and serial numbers.
+
+- :guilabel:`Consumable` products are items always assumed to be in stock, but with untracked and
+  unforecasted quantities (e.g., nails, toilet paper, and coffee). Consumables are replaceable and
+  essential, but exact counts are unnecessary
+- :guilabel:`Service`: perform and sell services (e.g., maintenance, installation, repair)
+
+.. image:: type/product-form.png
+   :align: center
+   :alt: Set a product type on the product form.
+
+.. note::
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, install the corresponding apps **in addition** to *Inventory*.
+
+   To install additional applications, navigate to :menuselection:`Apps` and select the
+   :guilabel:`Activate` button to install the desired apps or modules.
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Calendar* and
+  *Appointment* apps. Requires the :guilabel:`Pay to Book` module to be installed
```

Comment:
Hi @tiku-odoo ! What do you think of these descriptions of product types, as the calendar expert?

Reply:
This description looks fine. I need to familiarize myself with the pay-to-book module. You're good to go here, though. Booking of appointments would be through the appointments app (also the integration with website).

Reply:
```suggestion
- :guilabel:`Booking Fees`: charge a fee for booking appointments through the
  *Appointment* app. Requires the *Calendar* app and :guilabel:`Pay to Book` module to be installed
```